x-pack/plugins/cases/docs/openapi/components/schemas/connector_properties_resilient.yaml

```diff
@@ -0,0 +1,37 @@
+title: Create case request properties for a IBM Resilient connector
+required:
+  - fields
+  - id
+  - name
+  - type
+description: Defines properties for connectors when type is `.resilient`.
+type: object
+properties:
+  fields:
+    description: An object containing the connector fields. If you want to omit any individual field, specify null as its value.
+    type: object
+    nullable: true
+    required:
+      - issueTypes
+      - severityCode
+    properties:
```

Comment:
I believe `issueTypes` and `severityCode` are both string types (array of strings, and string that is).

https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/common/api/connectors/resilient.ts

Reply:
Thanks!  I've updated those data types in e07444fccc797dd4ea443e5113f366964a7d3078 and cc3b22002fa11a4b0108d69d4ebeed4a3ee33404